src/bin/solang.rs

```diff
@@ -185,6 +185,12 @@ fn main() {
                             .long("generate-debug-info")
                             .action(ArgAction::SetTrue)
                             .hide(true),
+                    )
+                    .arg(
+                        Arg::new("LOGRUNTIMEERRORS")
+                            .help("Log runtime errors in the environment")
+                            .long("log-runtime-errors")
+                            .action(ArgAction::SetTrue),
```

Comment:
This should probably be on by default, and a cli option to turn it off. What do you think @xermicus

Reply:
I think it should be on by default for ``revert(reason)`` and ``require(bool, reason)``

Reply:
As discussed in todays daily:
- Should behave the same as `--log-api-return-codes` (disabled by default)
- Should enable or disable everything related to runtime error reporting introduced in this PR, so it can be omitted completely for production environments